app/mixins/ads.js

```diff
@@ -40,6 +38,10 @@ export default Ember.Mixin.create({
 			config = adsData.additionalConfig[adSlotName] || {},
 			$placeholder = $('<div>');
 
+		if (!this.get('ads.module').isSlotApplicable(adSlotName)) {
```

Comment:
nitpick - could we do this check before assigning the values to vars above and querying the dom for the first div?